src/DynamoCore/Utilities/LuceneSearchUtility.cs

```diff
@@ -25,15 +26,40 @@
 
 namespace Dynamo.Utilities
 {
+    /// <summary>
+    /// Lucene search utility class that will be used for indexing and searching nodes and packages
+    /// </summary>
     internal class LuceneSearchUtility
     {
+        /// <summary>
+        /// Dynamo Model reference
+        /// </summary>
         internal DynamoModel dynamoModel;
+
+        /// <summary>
+        /// Index fields that were added to the document
+        /// </summary>
         internal List<string> addedFields;
+
+        /// <summary>
+        /// Lucene Directory Reader
+        /// </summary>
         internal DirectoryReader dirReader;
```

Comment:
this appears to be `Idisposable` as I am guessing some of the other lucene types are - do we dispose them somewhere?

Reply:
Yes, in `DynamoModel.Dispose()`